.release-please-manifest.json

```diff
@@ -1,18 +1,18 @@
 {
-  ".": "1.9.2",
-  "packages/api-explorer": "0.9.12",
+  ".": "1.10.0",
+  "packages/api-explorer": "0.9.13",
   "packages/code-editor": "0.1.5",
-  "packages/extension-api-explorer": "21.8.2",
-  "packages/extension-sdk": "21.8.3",
-  "packages/extension-sdk-react": "21.8.3",
-  "packages/hackathon": "21.8.2",
-  "packages/run-it": "0.9.12",
-  "packages/sdk": "21.8.3",
+  "packages/extension-api-explorer": "21.8.3",
+  "packages/extension-sdk": "21.8.4",
+  "packages/extension-sdk-react": "21.8.4",
+  "packages/hackathon": "21.8.3",
```

Comment:
dagnabit, these "dependency only" updates aren't picking up the parent `"release-as"` config, I'll troubleshoot why this afternoon but this is a blocker